spec/services/hyku_addons/cache_expiration_service_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+require "rails_helper"
+
+RSpec.describe HykuAddons::CacheExpirationService do
+  let(:service) { described_class.new }
+  let(:account) do
+    account = create(:account, name: 'tenant')
+    account.create_redis_endpoint!
+    account
+  end
+
+  before do
+    account
+  end
+
+  describe "perform" do
+    before do
+      allow(service).to receive(:expire_cache_for)
+      service.perform
+    end
+
+    it "updates the frontend_url of the Account" do
+      expect(service).to have_received(:expire_cache_for).once
```

Comment:
What does this have to do with the frontend URL?

Reply:
Copy pasta thing, good catch